java/ql/src/experimental/Security/CWE/CWE-665/InsecureRmiJmxEnvironmentConfiguration.qhelp

```diff
@@ -6,7 +6,7 @@
 <overview>
 <p>For special use cases some applications may implement a custom service which handles JMX-RMI connections.</p>
 
-<p>When creating such a custom service, a developer should pass a certain environment configuration to the JMX-RMI server initalisation, 
+<p>When creating such a custom service, a developer should pass a certain environment configuration to the JMX-RMI server initialisation, 
```

Comment:
```suggestion
<p>When creating such a custom service, a developer should pass a certain environment configuration to the JMX-RMI server initialization, 
```
(Since our style guide mandates the use of US English.)

Reply:
I'm willing to make this change (I personally favor en-US). Do note this repository has quite a few en-GB entries (including a number of other instances of `initialisation`). It'd probably be best, in _general_, if such corrections were done as a distinct PR (as is, this PR is intentionally scoped to just target `.qhelp` files).

I should really write a thing that makes it easy for me to flag en-GB or en-US entries -- it's something I haven't looked into in a while (mostly, I've decided to be skittish about this).

I'm going to toss in 3 related changes as a demo, and I'll tag them below. I can of course drop them / split them into another PR.

Reply:
I've made this change to the file, including renaming the two referenced java files. I have not applied the change to `initialization` for the rest of the repository.